server/spec/middlewares/websocket_backend_spec.rb

```diff
@@ -4,10 +4,23 @@
   let(:app) { spy(:app) }
   let(:subject) { described_class.new(app) }
 
+  before(:all) do
+    EM.run
```

Comment:
Basic pattern for running EM specs should be like on the agent: https://github.com/kontena/kontena/blob/master/agent/spec/lib/kontena/websocket_client_spec.rb#L13-L18

```
ruby
  around(:each, :em => true) do |example|
    EM.run {
      example.run
      EM.stop
    }
end
```